amr-wind/wind_energy/actuator/disk/disk_ops.cpp

```diff
@@ -384,6 +376,7 @@ void compute_disk_points(
         }
     }
 }
+} // namespace base
 } // namespace ops
 } // namespace actuator
-} // namespace amr_wind
+} // namespace amr_wind
```

Comment:
This shows a warning about there not being a line at the end of the file. Not sure if this matters.